css/style.css

```diff
@@ -40,4 +40,4 @@
 
 #tribute-link{
     color: #477ca7;
-}
\ No newline at end of file
+}/*linter error*/
```

Comment:
linter error has been found in line 43, there must be an empty space at the end of any css file for navigation and readability reasons, having no empty space in the end, makes your code look more compressed.

Reply:
- [Optional ] Kindly, just like last comment you given below phrased the comments politely, as suggestions instead of orders(above comment a little bit look like an order).